src/wallet/wallet2.cpp

```diff
@@ -908,6 +911,11 @@ uint8_t get_clsag_fork()
   return HF_VERSION_CLSAG;
 }
 
+uint8_t get_view_tag_fork()
```

Comment:
What's the idea behind this method, in contrast to directly use the constant in code?

Reply:
I'm not sure tbh, just figured makes sense to follow the pattern as with the others above. My guess is that you can change the name of the constant if needed more easily.. I don't know